### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by karenmajewicz**
Added documentation from community edited About page draft (https://docs.google.com/document/d/1YR15FKRbXoxuOS8cElyXyYJ5Cm1baElj0VhqiDIBnXM/edit?usp=sharing) and performed copyediting to make the page more web-friendly

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -1,30 +1,101 @@
...
+The GeoBlacklight Community is open to anyone interested in spatial infrastructures, libraries, GIS, maps, open source software, and related topics. Participants can contribute to the community in various ways, such as by attending meetings, writing documentation, developing metadata best practices, engaging in outreach, and writing code. Visit our [Connect page](https://geoblacklight.org/connect.html) to join our online community.
 
-### Community Values and Technical Values
+#### Events
 
-Members of the GeoBlacklight community are currently working on community values and technical values. These will be published soon.
+- **Every month:** [Zoom meetings](https://z.umn.edu/gbl-meetings) to share project updates and to discuss topical issues.
 
-### Project Roadmap
+- **2x per year:** [Community Sprints](https://github.com/geoblacklight/geoblacklight/projects?query=). These are similar to a traditional code sprint but also incorporates activities around documentation, metadata, governance, and more.
```
Comment (andrewbattista) [inline about.md:26]: Should be "These are similar to a traditional code spring but also incorporate..."

Comment (karenmajewicz) [reply about.md]: Thanks! I removed the 's'

File: about.md [hunk 2]
```diff
@@ -1,30 +1,101 @@
...
+	- records are formatted as JSON files
+
+**************************
+
+GeoBlacklight Technical Values
+------------------------------
+
+- Our core focus is **geospatial discovery**. This focus initially was limited to discretely catalogued data objects, but has expanded over time to include a wider range of information sources.
+
+- We emphasize **end-user experience**, including inclusivity and accessibility in design features whenever possible.
```
Comment (andrewbattista) [inline about.md:87]: I would put a comma after features in this sentence

**Review by andrewbattista — APPROVED** [review]
This looks great to me! Thanks, @karenmajewicz and anyone else who contributed

File: about.md (hunk 2 above)
Comment (karenmajewicz) [reply about.md]: Good note. I made an edit that just removes the clause "whenever possible" as it doesn't seem (to me) to be needed here.

File: about.md [hunk 3]
```diff
@@ -1,30 +1,101 @@
...
 layout: page
-title: About the GeoBlacklight Community
+title: About GeoBlacklight
 permalink: /about/
 ---
 
-- [Community Structure](#community-structure)
-- [Community Values and Technical Values](#community-values-and-technical-values)
-- [Project Roadmap](#project-roadmap)
+GeoBlacklight is a world-class discovery platform for geospatial content, including GIS data and maps. It is an open, multi-institutional collaborative project aiming to build off of the successes of the [Blacklight](https://projectblacklight.org/) discovery interface and the [OpenGeoMetadata](https://github.com/OpenGeoMetadata) federated metadata sharing communities.
```
Comment (mejackreed) [inline about.md:7]: This seems to have diverged from the draft. Could we align this better?

**Review by mejackreed — APPROVED** [review]
Thanks @karenmajewicz for moving this forward

**Comment by karenmajewicz** [comment]
OK, this should be good to review/merge again. (I had a bit of a runaround with a commit revert to remove something I inadvertently altered elsewhere). 

I changed the opening paragraph back to the original text from the external doc. I also added a link to OpenGeoMetadata further down the page.  Let me know how this looks now. Thanks!

**Comment by karenmajewicz** [comment]
Would someone be up for merging this? Thanks! (I think we might want to get it up before Geo4Lib)

**Pull request merged**